Corona-Warn-App/src/main/res/values-tr/strings.xml

```diff
@@ -1493,7 +1493,7 @@
     <!-- XTXT: Statistics explanation blog text -->
     <string name="statistics_explanation_blog">"Münferit önemli değerlerin içeriği hakkında daha ayrıntılı bilgi için blogumuzu okuyun."</string>
     <!-- XTXT: Statistics explanation blog text label -->
-    <string name="statistics_explanation_blog_label">"Blog."</string>
+    <string name="statistics_explanation_blog_label">"Blogumuzu"</string>
```

Comment:
is there a reason you deleted the . at the end?

Reply:
Yes. With the dot, the link replacement in the resource above would not work. The label `statistics_explanation_blog_label` gets replaced with the link in `statistics_explanation_blog`.